core/chains/evm/config/toml/defaults/Shibarium_Mainnet.toml

```diff
@@ -0,0 +1,38 @@
+ChainID = '109'
+FinalityTagEnabled = true
+LinkContractAddress = '0x71052BAe71C25C78E37fD12E5ff1101A71d9018F'
+LogPollInterval = '1s'
+MinIncomingConfirmations = 5
+NoNewHeadsThreshold = '30s'
+RPCBlockQueryDelay = 10
+RPCDefaultBatchSize = 100
+
+[Transactions]
+ResendAfterThreshold = '1m'
+
+[BalanceMonitor]
+Enabled = true
+
+[GasEstimator]
+EIP1559DynamicFees = true
+PriceDefault = '10 gwei'
+PriceMin = '2 gwei'
+BumpMin = '5 gwei'
+BumpThreshold = 5
+PriceMax = '10000 gwei'
+LimitDefault = 8000000
+FeeCapDefault = '10000 gwei'
+
+[GasEstimator.BlockHistory]
+# Default is 24, which leads to bumpy gas prices. In CCIP
+# we want to smooth out the gas prices, so we increase the sample size.
+BlockHistorySize = 200
```

Comment:
I noticed some of these configs are specific to CCIP. Since we're merging to the `chainlink` repo, do we need to update these or leave them as the default?

Reply:
Removed. Ty